doc/nrf/ug_thingy91.rst

```diff
@@ -171,50 +171,28 @@ Building and programming using VS Code extension
-
+#. In the Welcome panel or on the Welcome page, click :guilabel:`Add an existing application`.
    The :guilabel:`New Application` screen opens.
-
-#. Provide the following information to create the application:
-
-   a. Select the :guilabel:`Freestanding` radio button for the :guilabel:`Application type`.
-      This links the application to the version of |NCS| you have installed.
-   #. Select the |NCS| version from the :guilabel:`nRF Connect SDK` drop-down menu and the corresponding toolchain version from the :guilabel:`nRF Connect Toolchain` drop-down menu.
-      The version you selected when setting up |VSC| is set by default.
-   #. Choose the location where the application is saved.
-      To do this, either type the path in the :guilabel:`Application location` field, or browse to the desired location by clicking the :guilabel:`...` button.
-   #. Select the application template that you want to use.
-
-      You can either select one from the :guilabel:`Application template` drop-down menu, or browse the available templates with a preview of their documentation in the :guilabel:`Browse Samples` screen by clicking the :guilabel:`Select...` button.
-      Both the drop-down and the :guilabel:`Browse Samples` screen contain the same applications.
-   #. Enter the :guilabel:`Application name`.
-
-      When you create the application, this creates a new folder in the :guilabel:`Application folder` directory with the same name.
-
+#. Navigate to the directory of the nRF9160-compatible sample or application you want to work with.
+   Samples are located under :file:`nrf/samples/`.
+   Applications are located under :file:`nrf/applications/`.
+#. Click :guilabel:`Select folder` to select the sample's or the application's folder.
+   A new application is automatically added in the Applications panel.
 #. Click the :guilabel:`Create Application` button.
-
    The selected application appears in the :guilabel:`Applications` panel.
-
 #. Click the :guilabel:`No build configurations. *Click to create one*` text under the application.
-
    The :guilabel:`Add Build Configuration` screen opens.
-
 #. Provide the following information to configure the build:
 
    a. Select the build target from the :guilabel:`Board` drop-down menu.
-
       You must use the build target ``thingy91_nrf9160`` or ``thingy91_nrf9160_ns`` when building the application code for the nRF9160 SiP and the build target ``thingy91_nrf52840`` when building the application code for the onboard nRF52840 SoC.
    #. Select ``prj.conf`` from the :guilabel:`Configuration` drop-down menu if it is not already selected.
    #. Make sure that the checkbox for :guilabel:`Build after generating configuration` is selected.
 
 #. Click the :guilabel:`Build Configuration` button.
-
    The configuration starts building.
    It is complete when the build files appear in the :guilabel:`Applications` panel.
```

Comment:
```suggestion
   It is complete when the build files appear in the :guilabel:`build details` panel.
```

The build details panel isn't named that exactly, but this is how we refer to it. It usually has the name of the sample or application and then tiny text that says (build) - [example here](https://nrfconnect.github.io/vscode-nrf-connect/connect/build_app.html#setting-up-a-build-configuration). It is located under the Applications panel. I don't think it's necessary to state this, but I'm open to second opinions.

Reply:
It could be worded "...when the build files appear under the sample or application."

That would sidestep what to call that section.

Reply:
Reworked after discussion offline.